store/tikv/region_request.go

```diff
@@ -408,6 +411,11 @@ func (s *RegionRequestSender) onRegionError(bo *Backoffer, ctx *RPCContext, seed
 		logutil.BgLogger().Warn("tikv reports `RaftEntryTooLarge`", zap.Stringer("ctx", ctx))
 		return false, errors.New(regionErr.String())
 	}
+	if regionErr.GetRegionNotFound() != nil && seed != nil {
+		logutil.BgLogger().Debug("tikv reports `RegionNotFound` in follow-reader",
```

Comment:
Sorry to bother, How to make sure it is a `follower-read` here? 
need to check `req.ReplicaRead == true` to make sure it is a `follower-read`?

Reply:
:D we can make sure it by see if `seed == nil`， seed only be nil when it's follower read？